ui/app/AppLayouts/Chat/stores/MessageStore.qml

```diff
@@ -224,4 +242,23 @@ QtObject {
             return true
         return messageModule.didIJoinedChat();
     }
+
+    property bool playAnimation: {
+        if(!Global.applicationWindow.active)
+            return false
+
+        if(root.getSectionId() !== mainModule.activeSection.id)
+            return false
+
+        if(!root.chatSectionModule)
+            return false
+
+        if(root.chatSectionModule.activeItem.isSubItemActive &&
+                root.getChatId() !== root.chatSectionModule.activeItem.activeSubItem.id ||
+                !root.chatSectionModule.activeItem.isSubItemActive &&
+                root.getChatId() !== root.chatSectionModule.activeItem.id)
+            return false
+
+        return true
+    }
```

Comment:
I love scopes so if you can pls add :) but its my personal opinion so I just ask and its can be ignored :)

Reply:
@borismelnik that's a matter of preferences... I used to add `{}` if there is more than one line in `then`/`else` block, otherwise not.